app.go

```diff
@@ -638,6 +638,24 @@ func (app *App) GetRoute(name string) Route {
 	return Route{}
 }
 
+// GetRoutes Get all routes,when filterUseOption equal to true.it will filter the routes registered by the middleware.
```

Comment:
Is it possible to add spaces after punctuation and be case sensitive?

Reply:
done